src/covid_model_seiir_pipeline/forecasting/data.py

```diff
@@ -76,19 +88,28 @@ def check_covariates(self, scenarios: Dict[str, ScenarioSpecification]):
                 if not data_file.exists():
                     raise FileNotFoundError(f'No {covariate_version} file found for covariate {covariate}.')
 
+    def load_regression_coefficients(self, draw_id: int) -> pd.DataFrame:
+        return self.regression_marshall.load(MKeys.coefficient(draw_id))
+
+    # TODO: inverse is RegressionDataInterface.save_date_file
```

Comment:
Several methods have save/load pairs split between RegressionDataInterface and ForecastDataInterface. I've flagged them where the APIs don't align in these TODOs